app/javascript/components/activityorderhistory.vue

```diff
@@ -0,0 +1,64 @@
+<template lang="html">
```

Comment:
I made this a separate component instead of integrating this in `orderhistory.vue` because the isLoading and orders attributes would have to be arrays otherwise, which isn't very nice.